grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -162,7 +163,7 @@ public GraknAdmin admin(){
 
     @Override
     public <T extends Concept> T buildConcept(Vertex vertex) {
-        return elementFactory.buildConcept(vertex);
+        return getElementFactory().buildConcept(vertex);
```

Comment:
Initially the concept cache was in element factory and element factory was made thread local. This is just a left over of those few attempts. Is it that much of an issue that it needs to be changed ? I typically prefer using the getter rather than the field directly. `getTinkerTraversals()` shows why it is usually a good idea.